### PR TITLE
Configure CKEditor to work in inline mode #5129

### PR DESCRIPTION
**Pull request by ashklianko**
Missing features for now:
-source dialog; will be added in other issue
for both inline and classic modes:
-blur/focus handlers commented now; cke wraps natural events with their own events, need to investiage more on this because some listeners expect natural FocusEvent that not present in cke event
-keyboard events handling

### CODE REVIEW

**Comment by alansemenov** [comment]
* Multiple console errors whenever HTML area is clicked and gets focus
* Bottom margin of the toolbar should be the same as on the top
* "X" icon closing the edit mode should have 'pointer' cursor, not 'move'
* Remove margin inside the area (what I mentioned in #5130)

**Comment by ashklianko** [comment]
On it

**Comment by ashklianko** [comment]
Done

**Comment by ashklianko** [comment]
Had to add #5142 and #5132 in this PR , having issues with local build

File: modules/admin/admin-ui/src/main/resources/web/admin/common/js/form/inputtype/text/HtmlArea.ts [hunk 1]
```diff
@@ -237,6 +241,17 @@ module api.form.inputtype.text {
             });
         }
```
Comment (alansemenov) [inline modules/admin/admin-ui/src/main/resources/web/admin/common/js/form/inputtype/text/HtmlArea.ts]: Replace the two methods below with one called ``appendButtonToBottomBar``where you pass the identifying class of the button to move.

File: modules/admin/admin-ui/src/main/resources/web/admin/common/js/util/htmlarea/dialog/CharMapDialog.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 module api.util.htmlarea.dialog {
 
     import i18n = api.util.i18n;
+    import editor = CKEDITOR.editor;
```
Comment (alansemenov) [inline modules/admin/admin-ui/src/main/resources/web/admin/common/js/util/htmlarea/dialog/CharMapDialog.ts]: Change this to ``import HTMLAreaEditor = CKEditor.editor;`` (in all classes)-
It's a bit confusing when it says "``(editor: editor)``" in function signature.

Comment (ashklianko) [inline modules/admin/admin-ui/src/main/resources/web/admin/common/js/util/htmlarea/dialog/CharMapDialog.ts]: ok!

File: modules/admin/admin-ui/src/main/resources/web/admin/common/js/form/inputtype/text/HtmlArea.ts (hunk 1 above)
Comment (ashklianko) [inline modules/admin/admin-ui/src/main/resources/web/admin/common/js/form/inputtype/text/HtmlArea.ts]: Done

**Pull request merged**